Source/CesiumRuntime/Private/CesiumDmsEditor.cpp

```diff
@@ -0,0 +1,290 @@
+      GetDecimalDegreesFromProperty() < 0 ? NegativeIndicator
+                                          : PositiveIndicator);
+
+  const float hPad = 3.0;
+  const float vPad = 2.0;
+  // clang-format off
+	Row.CustomWidget().NameContent()
+		[
+			DecimalDegreesHandle->CreatePropertyNameWidget()
+		]
+		.ValueContent().HAlign(EHorizontalAlignment::HAlign_Fill)
+		[
+        SNew( SVerticalBox )
+        + SVerticalBox::Slot().Padding(0.0f, vPad)
+        [
+              DecimalDegreesSpinBox.ToSharedRef()
+        ]
+        + SVerticalBox::Slot().Padding(0.0f, vPad)
+        [
+            SNew( SHorizontalBox )
+            + SHorizontalBox::Slot().FillWidth(1.0)
+            [
+                DegreesSpinBox.ToSharedRef()
+            ]
+            + SHorizontalBox::Slot().AutoWidth().Padding(hPad, 0.0f)
+            [
+                // The 'degrees' symbol
+                SNew(STextBlock).Text(FText::FromString(TEXT("\u00B0")))
+            ]
+            + SHorizontalBox::Slot().FillWidth(1.0)
+            [
+                MinutesSpinBox.ToSharedRef()
+            ]
+            + SHorizontalBox::Slot().AutoWidth().Padding(hPad, 0.0f)
+            [
+                // The 'minutes' symbol
+                SNew(STextBlock).Text(FText::FromString(TEXT("\u2032")))
+            ]
+            + SHorizontalBox::Slot().FillWidth(1.0)
+            [
+                SecondsSpinBox.ToSharedRef()
+            ]
+            + SHorizontalBox::Slot().AutoWidth().Padding(hPad, 0.0f)
+            [
+                // The 'seconds' symbol
+                SNew(STextBlock).Text(FText::FromString(TEXT("\u2033")))
+            ]
+            + SHorizontalBox::Slot().AutoWidth()
+            [
+                SignComboBox.ToSharedRef()
+            ]
+        ]
+		];
+  // clang-format on
+}
+
+double CesiumDmsEditor::GetDecimalDegreesFromProperty() const {
+  double decimalDegrees;
+  FPropertyAccess::Result AccessResult =
+      DecimalDegreesHandle->GetValue(decimalDegrees);
+  if (AccessResult == FPropertyAccess::Success) {
+    return decimalDegrees;
+  }
+  // This should never be the case, if the actual prperty
+  // was a double property, so this warning indicates a
+  // developer error:
+  UE_LOG(LogCesium, Warning, TEXT("GetDecimalDegreesFromProperty FAILED"));
+  return decimalDegrees;
+}
+
+void CesiumDmsEditor::SetDecimalDegreesOnProperty(double NewValue) {
+  UE_LOG(
+      LogCesium,
+      VeryVerbose,
+      TEXT("SetDecimalDegreesOnProperty %f"),
+      NewValue);
+  DecimalDegreesHandle->SetValue(NewValue);
+
+  SignComboBox->SetSelectedItem(
+      NewValue < 0 ? NegativeIndicator : PositiveIndicator);
+}
+
+int32 CesiumDmsEditor::GetDegrees() const {
+  double decimalDegrees = GetDecimalDegreesFromProperty();
+  DMS dms = decimalDegreesToDms(decimalDegrees);
+  return static_cast<int32>(dms.d);
+}
+void CesiumDmsEditor::SetDegrees(int32 NewValue) {
+  UE_LOG(LogCesium, VeryVerbose, TEXT("SetDegrees %d"), NewValue);
```

Comment:
Log statements like this kind of just seem like clutter to me. If you really care about these events you could achieve the same thing without modifying the code by configuring the debugger to log for you:
![image](https://user-images.githubusercontent.com/924374/134346687-0b9a3b08-aa60-4e47-a44f-eff1ac50637c.png)
Is there some compelling purpose that isn't obvious to me?